<?xml version="1.0"?>
<launch>
    <include file="$(find assignment_2_2024)/launch/sim_w1.launch" />
    <param name="des_pos_x" value="0.0" />
    <param name="des_pos_y" value="1.0" />
    <node pkg="assignment_2_2024" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="assignment_2_2024" type="go_to_point_service.py" name="go_to_point" />
    <node pkg="assignment_2_2024" type="bug_as.py" name="bug_action_service" output="screen" />
    <node pkg="assignment_2_2024" type="request_last_goal.py" name="request_last_goal" output="screen" launch-prefix="gnome-terminal --geometry=50x24+0+500  -- bash -c" />
    <node pkg="assignment_2_2024" type="send_goal_client.py" name="send_goals" output="screen" launch-prefix="gnome-terminal --geometry=80x24+0+0  -- bash -c" />
    
    <node pkg="assignment_2_2024" type="echo_topic.sh" name="custom_msg_echo_node" output="screen" launch-prefix="gnome-terminal --geometry=80x24+1300+500  --" />

</launch>

